src/components/seo.js

```diff
@@ -54,59 +53,21 @@ const SEO = ({ description, lang, meta, title, twitterCard }) => {
   }
 
   return (
-    <Helmet
-      htmlAttributes={{
-        lang,
-      }}
-      title={title}
-      titleTemplate={`%s | ${site.siteMetadata.title}`}
-      meta={[
-        {
-          name: "description",
-          content: metaDescription,
-        },
-        {
-          property: "og:title",
-          content: title,
-        },
-        {
-          property: "og:description",
-          content: metaDescription,
-        },
-        {
-          property: "og:image",
-          content: site.siteMetadata.siteUrl + twitterCard,
-        },
-        {
-          property: "og:type",
-          content: "website",
-        },
-        {
-          name: "twitter:card",
-          content: "summary_large_image",
-        },
-        {
-          name: "twitter:creator",
-          content: site.siteMetadata.social.twitter,
-        },
-        {
-          name: "twitter:site",
-          content: site.siteMetadata.social.twitter,
-        },
-        {
-          name: "twitter:title",
-          content: title,
-        },
-        {
-          name: "twitter:description",
-          content: metaDescription,
-        },
-        {
-          name: "twitter:image",
-          content: site.siteMetadata.siteUrl + twitterCard,
-        },
-      ].concat(meta)}
-    />
+    <>
+      <title>{title}</title>
+      <meta name="description" content={metaDescription} />
+      <meta property="og:title" content={title} />
+      <meta property="og:description" content={metaDescription} />
+      <meta property="og:image" content={site.siteMetadata.siteUrl + twitterCard} />
+      <meta property="og:type" content="website" />
+      <meta name="twitter:card" content="summary_large_image" />
+      <meta name="twitter:creator" content={site.siteMetadata.social.twitter} />
+      <meta name="twitter:site" content={site.siteMetadata.social.twitter} />
+      <meta name="twitter:title" content={title} />
+      <meta name="twitter:description" content={metaDescription} />
+      <meta name="twitter:image" content={site.siteMetadata.siteUrl + twitterCard} />
+      <script src='//www.eclipse.org/eclipse.org-common/themes/solstice/public/javascript/vendor/cookieconsent/default.min.js' />
```

Comment:
I'd recommend using https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-script/ to add your script to your page. With its default handling you'll get better performance :)